### PR TITLE
Add new website pricing page

### PR DESCRIPTION
**Pull request by bmosoane**
## Summary
New pricing page on the website so that our clients can see the various pricing options we have before they contact us

## To ensure well tested
- [x] There is a new pricing page that works in mobile, tablet and desktop according to the designs
- [x] There is a german translation for the pricing page
- [x] There is a dutch translation for the pricing page
- [x] There is a link in the footer to the new pricing page
- [x] UTM gets attributed correctly when companies request an invite
- [x] The CTA on tech-hiring page is changed and points to the pricing page (as outlined in the description)

## Ticket
https://app.clickup.com/t/8mtzy6

### CODE REVIEW

**Comment by moekify** [comment]
Task linked: [CU-8mtzy6 Website new pricing](https://app.clickup.com/t/8mtzy6)

File: src/components/footer.js [hunk 1]
```diff
@@ -49,6 +49,13 @@ export default () => {
                 {t('header:tech-hiring')}
               </Link>
             </h5>
+            <ul className="footer__link-list">
+              <li className="footer__link-item">
+                <Link className="footer__link" to={`${prefix}tech-hiring/pricing`}>
+                  {t('pricing')}
```
Comment (jayjayjpg) [inline src/components/footer.js:55]: I'm not super familiar with React, so I might just leave a couple of curious questions here and there to learn more myself from your work, so feel free to answer them on your own time without blocking the merge of your PR - thank you!

Is `t` something like a translation helper here?

File: src/components/pricing/benefits.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import { NamespacesConsumer } from 'react-i18next'
+
+import { getLinkToWithUTM } from '../../utils/page'
+import links from '../sign-up-links.json'
+
+const benefits = ['benefits.1', 'benefits.2', 'benefits.3', 'benefits.4']
+const utm = ' ?utm_source=website&utm_medium=start%20hiring'
+
+const Benefits = ({ page }) => (
+  <NamespacesConsumer ns={page}>
```
Comment (jayjayjpg) [inline src/components/pricing/benefits.js:13]: What is the main reason for using the `NamespacesConsumer` component here?

**Comment by bmosoane** [comment]
reina: d website#bm/feature/pricing

**Review by jayjayjpg — COMMENTED** [review]
This reads great! I just left a few questions out of curiosity to learn more about your PR and hope someone with React expertise can help with some more interesting comments than mine :-)

File: src/components/pricing/benefits.js (hunk 2 above)
Comment (bmosoane) [reply src/components/pricing/benefits.js:13]: So we are using i18next for handling all the translations for our website.

The `NamespacesConsumer` is a render prop that allows us to pass information down to its child components (the translations) through the `t` function

Comment (bmosoane) [reply src/components/pricing/benefits.js:13]: You can read more about it here https://react.i18next.com/legacy-v9/namespacesconsumer

File: src/components/footer.js (hunk 1 above)
Comment (bmosoane) [reply src/components/footer.js:55]: Yes, it's a function that trigger loading the translation files that were defined. (Works within NamespacesConsumer)

**Review by jayjayjpg — APPROVED** [review]

File: src/components/pricing/benefits.js (hunk 2 above)
Comment (jayjayjpg) [reply src/components/pricing/benefits.js:11]: Thank yous!

**Pull request merged**